packages/widget-message-meet/README-USAGE.md

```diff
@@ -1,28 +1,109 @@
+The Spark Message and Meet widget allows developers to quickly and easily incorporate messaging through Cisco Spark into an application. This widget handles the heavy lifting of coordinating between your application and the Spark APIs, and provides a complete Spark UI and messaging experience without having to build all front end UI yourself.
 
-Depending on how comfortable you are with these frameworks, there are are a number of ways you can use the widget on your website and in your code.
+Our widget is built using React <https://github.com/facebook/react>, Redux <https://github.com/reactjs/redux>, and the Spark JS SDK <https://github.com/ciscospark/spark-js-sdk>. This allows a developer to extend the UI to support different messaging contexts.
 
-## Data Attribute
+## Table of Contents
+-   [Features](#features)
+-   [Getting Started](#getting-started)
+    -   [CDN](#cdn)
+    -   [Build from Source](#build-from-source)
+-   [Usage](#usage)
+    -   [HTML](#html)
+    -   [JSX](#jsx)
+-   [Browser Support](#browser-support)
 
-The easiest way to get the Spark Message Meet Widget into your web site is to add the built resources and attach data attributes to your a container.
+## Features
 
-1. Follow these instructions to checkout and build the SDK (https://github.com/ciscospark/spark-js-sdk/blob/master/README.md)
-1. From the root of the sdk repo, run the following to build the widget:
-```sh
-PACKAGE=widget-message-meet npm run grunt:package -- build
+Currently, this widget supports:
+-   Basic text based messaging
+-   Inline Markdown
+-   Sharing of files and documents
+-   Preview and download of files and documents
+-   Flagging messages for follow up
+
+## Getting Started
+
+Depending on how comfortable you are with these frameworks, there are are a number of ways you can "install" our code.
+
+### Spark for Developers
+
+If you haven't already, go to the Spark for Developers Portal (<https://developers.ciscospark.com>) and sign up for an account. Once you've created an account you can get your developer access token by clicking on your avatar at the top right of the screen.
+
+When you want to eventually create an integration and have your own users take advantage of the widget, you'll need to create an integration with the following scopes:
+
+  ```
+  spark:rooms_read
+  spark:rooms_write
+  spark:memberships_read
+  spark:memberships_write
+  spark:messages_read
+  spark:messages_write
+  ```
+
+Head over to the Spark for Developers Documentation for more information about how to setup OAuth for your app: <https://developer.ciscospark.com/authentication.html>
+
+### CDN
+
+Using our CDN requires the least amount of work to get started. To get started, add the following into your HTML file:
+```
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://code.s4d.io/widget-message-meet/main.css">
```

Comment:
Nope. Too many chances for a flash of unstyled views that I'd much rather do this. The other option I would be to compile those class names and inject inline styles, but that would bloat the js bundle.